drivers/remote/driver.go

```diff
@@ -146,6 +147,9 @@ func (d *driver) EndpointOperInfo(nid, eid types.UUID) (map[string]interface{},
 	if err := d.call("EndpointOperInfo", info, &res); err != nil {
 		return nil, err
 	}
+	if val, ok := res.Value["MacAddress"]; ok {
+		res.Value[netlabel.MacAddress], _ = net.ParseMAC(val.(string))
+	}
```

Comment:
The Value structure is defined as a generic `map[string]interface{}` without any specific API const to represent the Key. Hence assuming it as "MacAddress" seems unreasonable for the Drivers to adhere to (if not already).
OTOH, the netlabel constants are well defined and used for exactly such purpose of well defined strings that act as keys for these generic datatypes.
Hence, I think it should be driver's responsibility to have the Value filled with appropriate labels such as `netlabel.MacAddress` with the Mac-address rather than doing it at the remote driver mux point.

@tomdee WDYT ?

@mrjana @aboch Do you have any input ?